boards/jiminy-mega256rfr2/Makefile.features

```diff
@@ -0,0 +1,13 @@
+include $(RIOTBOARD)/common/arduino-atmega/Makefile.features
+
+# Put defined MCU peripherals here (in alphabetical order)
+
+
+
+# Various other features (if any)
+FEATURES_PROVIDED += periph_cpuid
```

Comment:
Isn't this already provided by the CPU makefile ? If yes, you can remove this line